core/social_auth.py

```diff
@@ -61,10 +64,73 @@ def login(request, payload: SocialAuthLoginSchema):
         oauth.save()
     token, _ = Token.objects.get_or_create(user=user)
     user_id = user.id
-    queryset = User.objects.prefetch_related("organizations").get(id=user_id)
-    logger.debug(queryset)
-    response = {
-        "auth_token" :token.key,
+    #HACK: Hardcoded everything as of now need to figure out a way to work this 
+    result = urlparse(os.environ["DATABASE_URL"])
+    username = result.username
+    password = result.password
+    database = result.path[1:]
+    hostname = result.hostname
+    port = result.port
+    conn = psycopg2.connect(user=username, password=password, host=hostname, port=port,database=database)
+    query = f'SELECT * FROM core_user_organizations WHERE user_id = {user_id}'
```

Comment:
why are social auth changes are in explore creation pr. Revert this change. This pr should only contain changes related to explore creation